recipes/rt-eqcorrscan/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "RT-EQcorrscan" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # and fall back to archive: 
+  # url: https://github.com/eqcorrscan/RT_EQcorrscan/archive/{{ version }}.tar.gz
+  sha256: d4a0edb8ef2a19bd5dc16b4acb7b73ebeb23187b788e012f12a53cf3b1acd0e8
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```

Comment:
numpy and setuptools can be removed

Reply:
`setuptools` removed, `numpy` seems to be required to build.